api/admin/routers/auth.js

```diff
@@ -4,7 +4,7 @@ const passport = require('../passport');
 const onSuccess = (req, res) => {
   const script = `
     <script>
-      window.opener.postMessage("success", "http://localhost:3000")
+      window.opener.postMessage("success", "*")
```

Comment:
We are not passing any important here, so setting the target (client) domain to "*". The client does verify that the message comes FROM the correct domain. I wanted to avoid a situation where the server is somehow dependent on the client for configuration.